### PR TITLE
[BLOOM] Update doc with more details

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

Addressing https://huggingface.co/bigscience/bloom/discussions/86 
I think that we should add the full list of the trained languages on the documentation so that we can refer to that whenever any user will have any question related to the trained languages

cc @ydshieh @Muennighoff

### CODE REVIEW

**Comment by younesbelkada** [comment]
cc @cakiki

File: docs/source/en/model_doc/bloom.mdx [hunk 1]
```diff
@@ -25,6 +25,13 @@ Several smaller versions of the models have been trained on the same dataset. BL
 - [bloom-7b1](https://huggingface.co/bigscience/bloom-7b1)
 - [bloom](https://huggingface.co/bigscience/bloom) (176B parameters)
 
+## Languages
+
+BLOOM has officially been trained in 46 languages. The full list of trained languages are:
```
Comment (ydshieh) [inline docs/source/en/model_doc/bloom.mdx:30]: Probably 

BLOOM has officially been trained in 46 human languages and X programming languages.

I am not sure if `officially` is necessary.

**Review by ydshieh — APPROVED** [review]
Nice to have this information, thank you! Please wait (an)other reviewer(s) approval(s).

**Comment by Muennighoff** [comment]
I would remove it, as it doesn't matter for the architecture, which these docs explain afaict? It's just an artefact of the data the models are trained on, hence it's arldy on the pretrained model & dataset READMEs. If someone were to release a BLOOM architecture model trained on different languages, this would just be confusing.

Also should probably say `Pre-trained BLOOM models were officially released in the following sizes:`, as theoretically it's available in whatever version/size someone wants, just need to train it from scratch

File: docs/source/en/model_doc/bloom.mdx (hunk 1 above)
Comment (younesbelkada) [reply docs/source/en/model_doc/bloom.mdx]: Great! Should be addressed in ed1e74972899d3f05c6886ea61c6f517c96f734e

**Comment by ydshieh** [comment]
Not sure if the model doc is restricted to the architecture.

- `T5` has `## Example scripts` section
- Some models include `training` and `generation` sections, for example, `t5` or `m2m_100`
- `Marian` has a `## Naming` section

But I agree that we can probably just have something similar to `Marian` doc


Note: The list of languages used in training can be found [here](link to model card page).

**Comment by younesbelkada** [comment]
Okay I agree with both points!
IMO we can just specify that this list of languages is only relevant for bloom models stated on the doc, and for custom bloom models one should refer to the corresponding model card.

**Pull request closed without merging**